### ISSUE TITLE
--help output is poor for flag "--ignore-version-check"

### ISSUE THREAD

**Issue by ryantig** [Files-com/files-cli#6]
files-cli --help output for the ignore-version-check flag is the exact same as "api-key", "API Key".
I think the ignore-version-check "API Key" replacement text would be something along the lines of "Do not check for newer version"

current output:
```
Flags:
      --api-key string         API Key
      --debug string           verbose logging
      --feature-flag strings   Enable feature flags
  -h, --help                   help for files-cli
      --ignore-version-check   API Key
  -o, --output string          file path to save output
      --profile string         setup a connection profile
      --reauthentication       If authenticating to the API via a session ID (as opposed to an API key), we require that you provide the session user’s password again in a X-Files-Reauthentication header for certain types of requests where we want to add an additional level of security. We call this process Reauthentication.
  -v, --version                version for files-cli
```

**Comment by Files-Michael** [comment]
Hello ryantig,

Thank you for taking the time to file this issue.

For the quickest investigation into this issue, please send an authenticated support ticket through the Files.com web interface. You can submit a support request using the Contact Support option in the top-right menu of the web interface. You access the menu by clicking on your user name.

**Comment by ryantig** [comment]
Fixed in 2.6.7 via commit [25a245c](https://github.com/Files-com/files-cli/commit/25a245c6e229764deb2a96e7cab195771365c529). Thanks!
